org.worldpossible.ScriptLauncher.yml

```diff
@@ -0,0 +1,35 @@
+app-id: org.worldpossible.ScriptLauncher
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+branch: eos3
+sdk: org.gnome.Sdk
+command: worldpossible-scriptlauncher
+
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # We need to be able to run commands on the host to execute scripts chosen by the user
+  - --talk-name=org.freedesktop.Flatpak
+  - --filesystem=host
+
+build-options:
+  env: { V: '1' }
+
```

Comment:
iirc, this is set by default by the sdk

Reply:
Okay, removed it. It was just set by the Builder GNOME app template.